zebra-state/src/service/check/anchors.rs

```diff
@@ -146,7 +146,7 @@ pub(crate) fn anchors_refer_to_earlier_treestates(
         // > (i.e. when there are any Action descriptions), MUST refer to some
         // > earlier block’s final Orchard treestate.
         //
-        // <https://zips.z.cash/protocol/protocol.pdf#actions>
+        // https://zips.z.cash/protocol/protocol.pdf#actions
```

Comment:
Minor/Optional: We don't need to change the links in normal comments that use `<`, `>`. The idea of normal comments is that the person writing it can do whatever they want. In this case, this are consensus comments so maybe we should keep them untouched ?

Reply:
I see the writer made markdown in this kind of consensus comments. Even if they are not rendered now because they are normal comments, they could be visible in the future.

Reply:
I think we can just merge as is. The reason why brackets were added in those comments was because we thought we would want to add them in all instances, in the future. Since we decided to not make them mandatory in regular comments, this seems fine now

Reply:
Yea, well, i don't think is very important but here is my reasoning about it. I think is ok to keep the formatted links here because this specific comments already uses `> Quote` for quoting text from the spec. This reveals markdown intention. If the text is markdown then it is suggested to use https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses

Reply:
I reverted some changes again in [398bebb](https://github.com/ZcashFoundation/zebra/pull/4494/commits/398bebb8b6a0b3a1682668a37292629a33fbd58d).

Reply:
Thanks!